src/ngraph/frontend/onnx_import/op/cum_sum.cpp

```diff
@@ -0,0 +1,57 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <memory>
+
+#include "cum_sum.hpp"
+#include "ngraph/op/cum_sum.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace op
+        {
+            namespace set_1
+            {
+                NodeVector cum_sum(const Node& node)
+                {
+                    auto inputs = node.get_ng_inputs();
+                    auto data = inputs.at(0);
+                    auto exclusive = node.get_attribute_value<std::int64_t>("exclusive", 0);
+                    auto reverse = node.get_attribute_value<std::int64_t>("reverse", 0);
+
+                    if (inputs.size() > 1)
+                    {
+                        auto axis = inputs.at(1); // optional input, 0-D tensor
+                        return NodeVector{
+                            std::make_shared<ngraph::op::CumSum>(data, axis, exclusive, reverse)};
+                    }
+                    else
+                    {
+                        int64_t axis = 0; // default
+                        return NodeVector{
+                            std::make_shared<ngraph::op::CumSum>(data, axis, exclusive, reverse)};
```

Comment:
This constructor has been removed from @pruthviIntel branch.
Please update your branch.

Reply:
I think this is the reason why your tests are failing now.